tests/manual/simple_bootloader/CMakeLists.txt

```diff
@@ -8,8 +8,6 @@ cmake_minimum_required(VERSION 3.20.0)
 
 include($ENV{ZEPHYR_BASE}/../sidewalk/samples/sid_end_device/bootloader_version.cmake)
 
-set(mcuboot_KCONFIG_ROOT $ENV{ZEPHYR_BASE}/../sidewalk/samples/sid_end_device/child_image/mcuboot/Kconfig.root)
-
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(simple_bootloader)
 
```

Comment:
scripts/generate_version_info_overlay.py and samples/sid_end_device/bootloader_version.cmake are also **NO**. Use a VERSION file

Reply:
Removed the script, but mcuboot can't find the version from file, it's always 0.0.0